svensk-exegetisk-arsbok.csl

```diff
@@ -761,7 +764,7 @@
       </choose>
     </layout>
   </citation>
-  <bibliography hanging-indent="true" et-al-min="10" et-al-use-first="9" subsequent-author-substitute="&#8212;&#8212;&#8212;" entry-spacing="0">
+  <bibliography hanging-indent="true" et-al-min="10" et-al-use-first="9" subsequent-author-substitute="———" entry-spacing="0">
```

Comment:
Both are correct, but we escape certain characters for repository styles that are hard to identify/distinguish by eye, like the various types of dashes and special (e.g. non-breaking) spaces.